src/main.py

```diff
@@ -1,17 +1,45 @@
 """The entry point of the application."""
 
-import sys
-
-from src.example import greeting
+import training_mode
+import db
+import os
+import game
+import stats
```

Comment:
A saját modulok elé írd oda, hogy `src.`
```suggestion
import src.db
import os
import src.game
import src.stats
```